src/main/kotlin/br/com/client_score/model/Client.kt

```diff
@@ -0,0 +1,21 @@
+package br.com.client_score.model
+
+import java.time.LocalDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+data class Client(
+
+    @Id
+    val id: UUID = UUID.randomUUID(),
+
+    @OneToOne
+    @JoinColumn(name = "person_id")
```

Comment:
Acredito que podemos adicionar a propriedade CascadeType ALL (quando acontecer alguma alteração no cliente, a pessoa será atualizada também)
Sugestão de conteúdo sobre cascade:
http://melissalobo.blogspot.com/2014/09/sobre-cascadetype.html

Reply:
Neste momento, não farei o uso do Cascade. Conforme o projeto for evoluindo, voltamos nesse ponto.